### ISSUE TITLE
AZSoundManager crash with artwork

### ISSUE THREAD

**Issue by matelo** [zzzunov/AZSoundManager#4]
In `loadMetadata` method, runtime finds artwork key. Then it jumps `AVMetadataKeySpaceID3` condition. However, artwork is maybe somehow corrupted, or the format is not supported (even when iTunes is able to display it) and I get app crash with following exception:

`2016-10-24 16:40:39.700 Fotbalpool[18473:895860] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSCFData objectForKey:]: unrecognized selector sent to instance 0x61000005f7a0'
*** First throw call stack:
(
    0   CoreFoundation                      0x000000011032d34b __exceptionPreprocess + 171
    1   libobjc.A.dylib                     0x0000000114dbf21e objc_exception_throw + 48
    2   CoreFoundation                      0x000000011039cf34 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
    3   CoreFoundation                      0x00000001102b2c15 ___forwarding___ + 1013
    4   CoreFoundation                      0x00000001102b2798 _CF_forwarding_prep_0 + 120
    5   Fotbalpool                          0x000000010dce2526 __27-[AZSoundItem loadMetadata]_block_invoke + 902
    6   Fotbalpool                          0x000000010dce2094 -[AZSoundItem loadMetadata] + 1140
    7   Fotbalpool                          0x000000010dce19a6 __37-[AZSoundItem initWithContentsOfURL:]_block_invoke + 38
    8   libdispatch.dylib                   0x0000000115871980 _dispatch_call_block_and_release + 12
    9   libdispatch.dylib                   0x000000011589b0cd _dispatch_client_callout + 8
    10  libdispatch.dylib                   0x000000011587a366 _dispatch_queue_override_invoke + 1426
    11  libdispatch.dylib                   0x000000011587c3b7 _dispatch_root_queue_drain + 720
    12  libdispatch.dylib                   0x0000002016-10-24 16:40:39.701202 Fotbalpool[18473:895435] subsystem: com.apple.coreaudio, category: aqsrv, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 2, enable_private_data: 0
011587c08b _dispatch_worker_thread3 + 123
    13  libsystem_pthread.dylib             0x0000000115c4a746 _pthread_wqthread + 1299
    14  libsystem_pthread.dylib             0x0000000115c4a221 start_wqthread + 13
)
2016-10-24 16:40:39.701654 Fotbalpool[18473:931065] [] __tcp_connection_write_eof_block_invoke Write close callback received error: [89] Operation canceled
libc++abi.dylib: terminating with uncaught exception of type NSException`

Basically, metadataItem looks like this:

```
Printing description of metadataItem:
<AVMetadataItem: 0x610000015a30, identifier=id3/APIC, keySpace=org.id3, key class = NSTaggedPointerString, key=APIC, commonKey=artwork, extendedLanguageTag=(null), dataType=com.apple.metadata.datatype.JPEG, time={INVALID}, duration={INVALID}, startDate=(null), extras={
    dataType = "image/jpg";
    dataTypeNamespace = "org.iana.media-type";
    info = "";
    pictureType = "Cover (front)";
}, value length=31463>
```
